src/detail-page/components/LearnList.tsx

```diff
@@ -0,0 +1,79 @@
+import Button from '@material-ui/core/Button';
+import FiberManualRecordIcon from '@material-ui/icons/FiberManualRecord';
+import React from 'react';
+import styled from 'styled-components';
+import { ProfessionId } from '../../data/home-page/ITProfessions';
+import { capitalizeFirst, getData } from './LearnList.helpers';
+
+interface LearnListProps {
+    professionId: ProfessionId;
+  }
+
+const StyledDiv = styled('div')`
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    align-items: flex-start;
+`;
+
+const StyledButton = styled(Button)`
+    border-radius: 0;
+    font-weight: bold;
+    text-transform: none;
+    border: 2px solid ${props => props.theme.palette.primary.main};
+    margin-bottom: 1rem;
+
+    &:hover {
+        border: 2px solid ${props => props.theme.palette.primary.main};
+    }
+`;
+
+const StyledList = styled('ul')`
+    list-style: none;
+    margin-top: 0;
+    margin-bottom: 1rem;
+    padding-left: 0;
+`;
+
+const StyledLi = styled('li')`
+    display: flex;
+    line-height: 1.2;
+
+    &:not(:first-child) {
+      margin: .8rem 0;
+    }
+
+    p {
```

Comment:
We can extract the `p` as a separate styled component in order to reduce css nesting (and the related complexity) as much as possible! :muscle:

Reply:
Done :)